src/Netsphere.Network/Message/Game/C2S.cs

```diff
@@ -62,7 +62,7 @@ public class CLoginReqMessage : IGameMessage
     public class CQuickStartReqMessage : IGameMessage
     {
         [BlubMember(0)]
-        public byte Unk { get; set; }
+        public byte GameRule { get; set; }
```

Comment:
Use the [enum](https://github.com/wtfblub/NetspherePirates/blob/dotnetty/src/Netsphere/Constants.cs#L154) and make sure to the data type is right for serialization. [Example](https://github.com/wtfblub/NetspherePirates/blob/7fc82bc5216eb34416cb8af4a2d804ac9de0d126/src/Netsphere.Network/Data/Game/RoomDto.cs#L19)